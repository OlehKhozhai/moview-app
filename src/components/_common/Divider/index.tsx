import React from 'react';
import cn from 'classnames';

import styles from './styles.module.scss';

type DividerProps = {
  className?: string;
};

const Divider: React.FC<DividerProps> = ({ className }) => {
  return <hr className={cn(styles.root, className)} />;
};

export default React.memo(Divider);
