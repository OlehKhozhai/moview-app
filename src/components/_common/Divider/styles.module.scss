@import 'assets/styles/mixins/colors';

.root {
  width: 100%;
  height: 6px;
  background-color: $primary-light-bg;
  border: none;
}
