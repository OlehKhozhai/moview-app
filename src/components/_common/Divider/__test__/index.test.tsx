import React from 'react';
import { create } from 'react-test-renderer';

import Divider from '../index';

describe('Divider component', () => {
  test('should render Divider component by default', () => {
    const component = create(<Divider />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
