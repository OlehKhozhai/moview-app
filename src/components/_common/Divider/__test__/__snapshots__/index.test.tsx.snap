// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider component should render Divider component by default 1`] = `
<hr
  className="root"
/>
`;
